apps/dashboard/app/javascript/packs/batch_connect_templates_prefill.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+import Swal from 'sweetalert2'
```

Comment:
Can we not use this library? I'd like to to get rid of it at some point - https://github.com/OSC/ondemand/issues/2592

Reply:
Do you have any suggestions for the replacement of that library? (I don't think it will be a good idea to reinvent the wheels to create a modal library in this PR...)

Reply:
`window.alert`? I'd have to check the accessibility of bootstrap modals, but that'd be another option depending on how screen readers deal with them.

Reply:
Just switched to bootstrap modals. Things like `window.alert` won't suit our user interaction case here.